### PR TITLE
Support for Quorum private transaction payload

### PR DESCRIPTION
**Pull request by ziegfried**

### CODE REVIEW

File: config.schema.json [hunk 1]
```diff
@@ -38,6 +38,10 @@
                     "description": "Max. number of blocks to fetch at once",
                     "type": "number"
                 },
+                "decryptPrivateTransactions": {
+                    "description": "For chains/nodes that do support private transcations, this settings instructs block watcher to\nattempt to load the decrypted payload for private transactions",
```
Comment (atoulme) [inline config.schema.json:42]: transcations, typo

Comment (atoulme) [reply config.schema.json:42]: this settings -> this setting

File: src/blockwatcher.ts [hunk 2]
```diff
@@ -300,9 +296,36 @@ export class BlockWatcher implements ManagedResource {
...
+                    debug('Retrieving decrypted private transaction input %o', {
+                        txHash: rawTx.hash,
+                        input: rawTx.input,
+                    });
+                    try {
+                        const input = await this.nodePlatform.getRawTransactionInput(rawTx.input, this.ethClient);
+                        debug('Successfully retrieved decrypted transaction payload %o', { input });
+                        privatePayload = { input };
+                    } catch (e) {
+                        error('Failed to retrieve decrypted transaction payload %o', {
```
Comment (atoulme) [inline src/blockwatcher.ts:312]: any chance to log e?

File: src/config.ts [hunk 3]
```diff
@@ -197,6 +197,11 @@ export interface BlockWatcherConfigSchema {
     startAt: StartBlock;
     /** Wait time before retrying to fetch and process blocks after failure */
     retryWaitTime: WaitTimeConfig;
+    /**
+     * For chains/nodes that do support private transcations, this settings instructs block watcher to
```
Comment (atoulme) [inline src/config.ts:201]: typo

File: src/blockwatcher.ts (hunk 2 above)
Comment (ziegfried) [reply src/blockwatcher.ts]: good call!

**Pull request merged**